.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = false
```

Comment:
question: Why that? Those newlines are good, in my opinion

Reply:
Oups, agree with you. Copied the config from one of the projects, and here's the result